services/tasks/local_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
`_unix.go` doesn't work?

Reply:
there is no such GOOS

Reply:
👌

Reply:
@fuweid - I did that. I don't know the proper name for "*nix" IE: anything but windows. I haven't found a good pattern people use for that type of naming yet. Ideas?